marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -189,7 +189,7 @@ class Jobs {
       docObj.batch.timeEnded = fn.currentDateTime();
     }
     if(error){
-      let stackTraceObj = error.stackTrace[0];
+      let stackTraceObj = error.stackFrames[0];
       docObj.batch.fileName = stackTraceObj.uri;
```

Comment:
@ryanjdew ,
'stackTrace' seems to be undefined in 'error' object. did you mean it to be 'stackFrames' as I see it has uri and line ?

Reply:
@srinathgit Thanks for catching that.